README.md

```diff
@@ -39,6 +40,11 @@ linked Docker containers, run the following:
 docker-compose up
 ```
 
+## Access
+
+yarnitor will be accessible through localhost:XXXX where XXXX is whatever was set in the
```

Comment:
It might be on localhost, but not always. It's really going to be on whatever host name your docker engine is running (e.g., 192.168.100.99 if you're using Docker Toolkit with docker-machine).

So maybe just say it'll be accessible on http://<your docker host name>:XXXX where XXXX is the EXPOSED port.